docs/tutorials/projects/verification-system.md

```diff
@@ -0,0 +1,306 @@
+| `verification_confidence` | float | Optional | The detection confidence needed in order to move into the `DISARMED` state. <br> Default: `0.8` |
+| `countdown_time_s` | int | Optional | The time in seconds the system will remain in state `COUNTDOWN` before transitioning to state `ALARM`. <br> Default: `20` |
+| `alarm_time_s` | int | Optional | The time in seconds the system will remain in  state `ALARM` before transitioning to state `TRIGGER_1`. <br> Default: `10` |
+| `disarmed_time_s` | int | Optional | The time in seconds the system will remain in  state `DISARMED` before transitioning to state `TRIGGER_1`. <br> Default: `10` |
+| `disable_alarm` | bool | Optional | Disables the `COUNTDOWN` and `ALARM` states. The system will always remain in the `TRIGGER_1` and `TRIGGER_2` states. <br> Default: `false` |
+
+Note that `ALARM` is by default silent and only a label overlay, but you can [poll the classifications](/ml/vision/#getclassificationsfromcamera) to set up a physical alarm trigger.
+
+## Configure a transform camera
+
+To view the classifications that your verification system makes, configure a transform camera.
+
+Navigate to the **Config** tab of your machine’s page in [the Viam app](https://app.viam.com/).
+Click on the **Components** subtab and click **Create component**. Select the `camera` type, then select the `transform` model. Enter a name for your camera and click **Create**.
+
+With **Builder** mode selected, copy and paste the following attributes JSON into your camera's **Attributes** box:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "pipeline": [
+    {
+      "type": "classifications",
+      "attributes": {
+        "classifier_name": "security",
+        "confidence_threshold": 0.5
+      }
+    }
+  ],
+  "source": "my-webcam"
+}
+```
+
+Note that the camera name in `"source"` and the `"classifier_name"` match the names that we configured for our source camera and verification system service, respectively.
+If you used different names, edit as applicable.
+View these [instructions to configure a transform camera](/components/camera/transform/) for reference.
+Save your config.
+
+## View your verification system in action
+
+Navigate to the **Control** tab.
+Expand the card matching the name of the transform camera you configured to view an image stream with the system's notifications overlaid:
+
+![Verification camera feed](/tutorials/verification-system/disarmed.png)
+
+## Next Steps
+
+Now that you've got the verification aspect of your system working, you can use this as a launch point for customizing your own DIY home security system.
+For example:
+
+- Try making it so your machine triggers a physical alarm when the `ALARM` classification is emitted. Write a program with a [Viam SDK](/sdks/) and use [`GetClassificationsFromCamera()`](/ml/vision/#getclassificationsfromcamera) to poll for the classifications made.
```

Comment:
This is the major purpose of this module: to make a sound or light up an LED screen or dial 911 when a particular state is reached. I agree with Naomi that we need more here to bring that home. Here's a rough take:


```suggestion
- Write a program using one of the [Viam SDK](/sdks/) to poll the `facial-verification` module for its current state, and take action when a particular state is reached.
   For example, you could use [`GetClassificationsFromCamera()`](/ml/vision/#getclassificationsfromcamera) to capture when a transition into the `ALARM` state occurs, and then send you an Email with the captured image of the trespasser!
```